examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -301,6 +306,25 @@ static sl_status_t wfx_rsi_init(void)
         SILABS_LOG("sl_wifi_get_mac_address failed: %x", status);
         return status;
     }
+#ifdef SIWX_917
+    uint32_t trngKey[TRNGKEY_SIZE] = { 0x16157E2B, 0xA6D2AE28, 0x8815F7AB, 0x3C4FCF09 };
```

Comment:
could/should this be a `constexpr`

Reply:
These are any 4 random values to induce entropy to random generator. Yes, we can make this as constexpr.